validator_manager/src/list_validators.rs

```diff
@@ -0,0 +1,206 @@
+use std::path::PathBuf;
+
+use clap::{App, Arg, ArgMatches};
+use eth2::SensitiveUrl;
+use serde::{Deserialize, Serialize};
+
+use crate::{common::vc_http_client, DumpConfig};
+
+pub const CMD: &str = "list";
+pub const VC_URL_FLAG: &str = "vc-url";
+pub const VC_TOKEN_FLAG: &str = "vc-token";
+
+pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
+    App::new(CMD)
+    .about("Lists all the validators in VC")
+    .arg(
+        Arg::with_name(VC_URL_FLAG)
+        .long(VC_URL_FLAG)
+        .value_name("HTTP_ADDRESS")
+        .help(
+            "A HTTP(S) address of a validator client using the keymanager-API. \
+            If this value is not supplied then a 'dry run' will be conducted where \
+            no changes are made to the validator client.",
+        )
+        .default_value("http://localhost:5062")
+        .requires(VC_TOKEN_FLAG)
+        .takes_value(true)
+    )
+    .arg(
+        Arg::with_name(VC_TOKEN_FLAG)
+            .long(VC_TOKEN_FLAG)
+            .value_name("PATH")
+            .help("The file containing a token required by the validator client.")
+            .takes_value(true),
+    )
+}
+
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct ListConfig {
+    pub vc_url: SensitiveUrl,
+    pub vc_token_path: PathBuf,
+}
+
+impl ListConfig {
+    fn from_cli(matches: &ArgMatches) -> Result<Self, String> {
+        Ok(Self {
+            vc_token_path: clap_utils::parse_required(matches, VC_TOKEN_FLAG)?,
+            vc_url: clap_utils::parse_required(matches, VC_URL_FLAG)?,
+        })
+    }
+}
+
+pub async fn cli_run<'a>(
+    matches: &'a ArgMatches<'a>,
+    dump_config: DumpConfig
+) -> Result<(), String> {
+    let config = ListConfig::from_cli(matches)?;
+    if dump_config.should_exit_early(&config)? {
+        Ok(())
+    } else {
+        println!("{}", run(config).await.unwrap());
```

Comment:
This fixes the panic bug in no. 1 here: #5347 (comment)

Thanks to @michaelsproul, he also mentioned that we should remove all unwrap() in the PR

```suggestion
        println!("{}", run(config).await?);
```

Reply:
Fixed in [b80d16e](https://github.com/sigp/lighthouse/pull/5347/commits/b80d16eefcc0e395771ab42eede3079d5a34d2af)